### PR TITLE
complete cc8 all except writing test

### PR DESCRIPTION
**Pull request by simon-panek**

### CODE REVIEW

**Comment by simon-panek** [comment]
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

File: javascript/code-challenges/linked-list/linkedList/index.js [hunk 1]
```diff
@@ -103,6 +103,34 @@ class LinkedList { //creates LinkedList class
...
+        currentNodeOne = currentNodeOne.next;
+      }
+      currentNodeOne = currentNodeOne.next;
+      currentNodeTwo = currentNodeTwo.next;
+    }
+
+    return givenListOne;
+
+  }
+
```
Comment (adrianhuebner) [inline javascript/code-challenges/linked-list/linkedList/index.js:133]: I would like to see some tests added
